config/local.yml

```diff
@@ -19,6 +18,11 @@ log:
 #         interval: 2h
 #         grace_period: 48h
 
+audit:
```

Comment:
having this enabled by default for running locally will hopefully make it easier to catch any potential issues going forward